src/interface/state_descriptor.hpp

```diff
@@ -140,13 +141,6 @@ class StateDescriptor {
     return true;
   }
 
-  void AddMeshBlockPack(const std::string &pack_name, const VarPackingFunc<Real> &func) {
-    realVarPackerMap_[pack_name] = func;
-  }
-  void AddMeshBlockPack(const std::string &pack_name, const FluxPackingFunc<Real> &func) {
-    realFluxPackerMap_[pack_name] = func;
-  }
-
```

Comment:
is this branch based on Phil Grette's branch or something? Should we merge that in first? To make sure things are consistent? I thought we already removed this stuff.

Reply:
I just don't think we completely scrubbed it.